docs/tutorials/getting-started/go.mdx

```diff
@@ -0,0 +1,243 @@
+---
+title: Go SDK and Go Gin
+description: Getting Started with the OpenFeature Go SDK and Go Gin
+---
+
+import FlagdContent from '@site/src/components/custom/tutorial/flagd-content.mdx';
+import FlagdChangeContent from '@site/src/components/custom/tutorial/flagd-change-content.mdx';
+import WhyDefaultContent from '@site/src/components/custom/tutorial/why-default-content.mdx';
+
+# Getting Started with the OpenFeature Go SDK and Go Gin
+
+## Introduction
+
+This walk-through teaches you the basics of using OpenFeature in Go in the context of a Go Gin web application.
+
+You'll learn how to:
+
+- Integrate the OpenFeature Go SDK
+- Install and configure the OpenFeature provider
+- Perform basic feature flagging
+
+## Requirements
+
+## Walk-through
+
+This walk-through assumes that:
+
+- You have a basic knowledge of Go
+- You have Go 1.17 or later
+- You have Docker installed and running on the host system
+
+### Step 1: Create a new Go project
+
+To get started, let's create a new folder and initialize the project. This can be done by running following commands.
+
+```shell
+mkdir GoStart
+cd GoStart
+go mod init GoStart
```

Comment:
I've not seen a go module include capitalised letters before, could we use lowercase?